test/bindings/amqp/0.2.0/message/extended.json

```diff
@@ -0,0 +1,10 @@
+{
+    "contentEncoding": "gzip",
+    "messageType": "user.signup",
+    "bindingVersion": "0.2.0",
+    "x-number": 0,
+    "x-string": "",
+    "x-object": {
+        "property": {}
```

Comment:
are empty properties supposed to be serialized?
From a specification perspective, it doesnt make much sense, as this isnt actually communicating much in terms of that property.